docs/self-managed/identity/user-guide/connect-to-an-existing-keycloak.md

```diff
@@ -0,0 +1,65 @@
+---
+id: connect-to-an-existing-keycloak
+title: "Connect to an existing Keycloak instance"
+sidebar_label: "Connect to an existing Keycloak instance"
+---
+
+In this guide, we'll demonstrate how to connect the Identity component to your existing Keycloak instance.
+
+### Prerequisites
+
+- Access to your Keycloak Admin Console
+- A basic understanding of administering realms and clients in Keycloak
```

Comment:
Do you have resources here you could link to? For example, a link to [Keycloak Admin Console] and a link to documentation on [administering realms and clients in Keycloak].